### PR TITLE
Feature/v1 1 23 docs

### PR DESCRIPTION
**Pull request by mfahampshire**
* fixes broken links from `mdbook last-changed` plugin 
* includes commits from https://github.com/nymtech/nym/pull/3602 
* fixes broken links in readme to platform and wallet build instructions 
* adds tutorial on using element with the mixnet

### CODE REVIEW

File: documentation/dev-portal/src/tutorials/matrix.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Command-line**
+
+To start Matrix's Element client via Socks5 proxy connected to NymConnect open terminal and run:
+
+```sh
+element-desktop --proxy-server=socks5://127.0.0.1:1080
+```
+
+**Keybinding**
```
Comment (mfahampshire) [inline documentation/dev-portal/src/tutorials/matrix.md]: I'd split this into 2: the current one is the GUI option, and then we can add a CLI option with making an `alias` ?

File: documentation/dev-portal/src/tutorials/monero.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Monero NymConnect Integration
```
Comment (mfahampshire) [inline documentation/dev-portal/src/tutorials/monero.md:1]: why is this file in here again, its already deployed to prod

File: documentation/dev-portal/src/tutorials/matrix.md (hunk 1 above)
Comment (serinko) [reply documentation/dev-portal/src/tutorials/matrix.md]: Maybe I explain wrong. Both of these are GUI Element options, just one starts with a cmd and one initiate the same cmd with pressing keys. It's a nice way around.

Comment (serinko) [reply documentation/dev-portal/src/tutorials/matrix.md]: Feel free to change it though, I know it's not a good time to discuss back and forth.

**Pull request merged**